### PR TITLE
Adoption fixes

### PR DESCRIPTION
**Pull request by bdryanovski**
A lot of fixes related to the adoption guide code snippets

* let both left and right sides of the snippets match as much as possible and keep the focus on arguments and code structure
* fixing broken snippets that were not working inside Stackblitz
* remove unneeded code

### CODE REVIEW

File: src/app/migrations/datalist/datalist.2.core.txt [hunk 1]
```diff
@@ -5,30 +5,37 @@ import { FormGroup, FormControl, Validators } from '@angular/forms';
   selector: 'app-root',
   template: `
     <form clrForm [formGroup]="model">
-      <cds-datalist status="{{ itemHasErrors ? 'error' : 'neutral' }}" control-width="shrink">
-        <label>datalist</label>
+      <cds-datalist control-width="shrink">
+        <label>Reactive Datalist</label>
```
Comment (bbogdanov) [inline src/app/migrations/datalist/datalist.2.core.txt:9]: What is the intention behind this change?

File: src/app/migrations/range/range.1.core.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
```
Comment (bbogdanov) [inline src/app/migrations/range/range.1.core.txt:1]: Range and Select are not merged yet. Do you plan to merge this PR after the `range` and `select` are merged?

File: src/app/migrations/toggle/toggle.1.angular.txt [hunk 3]
```diff
@@ -5,8 +5,8 @@ import { Component } from '@angular/core';
   template: `
     <form clrForm>
       <clr-toggle-wrapper>
-        <input type="checkbox" clrToggle name="input" checked />
         <label>toggle</label>
+        <input type="checkbox" clrToggle name="input" checked />
```
Comment (bbogdanov) [inline src/app/migrations/toggle/toggle.1.angular.txt:9]: What is the intention behind this change?

File: src/app/migrations/toggle/toggle.2.angular.txt [hunk 4]
```diff
@@ -6,10 +6,10 @@ import { Component } from '@angular/core';
     <form clrForm>
       <clr-toggle-container>
         <clr-toggle-wrapper>
-          <input type="checkbox" clrToggle name="input" [(ngModel)]="input" />
           <label>toggle</label>
+          <input type="checkbox" clrToggle name="input" [(ngModel)]="input" />
         </clr-toggle-wrapper>
-        <clr-control-helper>message text</clr-control-helper>
+        <clr-control-helper>Мessage text</clr-control-helper>
```
Comment (bbogdanov) [inline src/app/migrations/toggle/toggle.2.angular.txt:12]: Is this necessary? Having in mind that both sides of the example are looking the same

File: src/app/migrations/toggle/toggle.2.core.txt [hunk 5]
```diff
@@ -7,7 +7,7 @@ import { Component } from '@angular/core';
       <cds-toggle>
         <label>toggle</label>
         <input type="checkbox" checked [(ngModel)]="input" />
-        <cds-control-message>message text</cds-control-message>
+        <cds-control-message>Меssage text</cds-control-message>
```
Comment (bbogdanov) [inline src/app/migrations/toggle/toggle.2.core.txt:10]: Same as the one above.

File: src/app/pages/badge.page.ts [hunk 6]
```diff
@@ -14,8 +14,12 @@ import { DemoTabData } from '../components/demo.component';
 
      <app-eslint-intro-block rule="no-clr-badge"></app-eslint-intro-block>
 
-    <demo [tabs]="demo1"></demo>
-    <demo [tabs]="demo2"></demo>
+    <demo [tabs]="demo1">
+      <h2>Basic</h2>
```
Comment (bbogdanov) [inline src/app/pages/badge.page.ts:18]: Thank you for adding these!

**Review by bbogdanov — CHANGES_REQUESTED** [review]

File: src/app/migrations/datalist/datalist.2.core.txt (hunk 1 above)
Comment (bdryanovski) [reply src/app/migrations/datalist/datalist.2.core.txt:9]: Simplification and most of all working inside stackblitz. Also try to keep both side equal so the focus will be on the code flow and attributes

File: src/app/migrations/range/range.1.core.txt (hunk 2 above)
Comment (bdryanovski) [reply src/app/migrations/range/range.1.core.txt:1]: You are right they slipped (bad fork) - will merge this PR after I merge the other one.

File: src/app/migrations/toggle/toggle.1.angular.txt (hunk 3 above)
Comment (bdryanovski) [reply src/app/migrations/toggle/toggle.1.angular.txt:9]: The left and right sides are not equal .. and there is no reason for the order of the HTML elements but could make someone believe there is.

`ClrToggleWrapper` will find and pick the elements inside its internal template so again there is no reason for reordering but this way both sides look identical and also try to keep the semantic of the HTML clear

File: src/app/migrations/toggle/toggle.2.angular.txt (hunk 4 above)
Comment (bdryanovski) [reply src/app/migrations/toggle/toggle.2.angular.txt:12]: same as above

File: src/app/migrations/toggle/toggle.2.core.txt (hunk 5 above)
Comment (bdryanovski) [reply src/app/migrations/toggle/toggle.2.core.txt:10]: same as above, above

File: src/app/pages/badge.page.ts (hunk 6 above)
Comment (bdryanovski) [reply src/app/pages/badge.page.ts:18]: For nothing

**Review by bbogdanov — APPROVED** [review]

**Review by mathisscott — APPROVED** [review]
Assuming @bbogdanov 's concerns are addressed... 👍

**Pull request merged**